### PR TITLE
always override auto labels

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/pre_add_labels_in_df.py [hunk 1]
```diff
@@ -162,7 +166,7 @@ def generate_auto_labels(self, base_labels, df_labels, plugin_labels):
             if lbl not in generated:
                 self.log.warning("requested automatic label %r is not available", lbl)
 
-            elif lbl in plugin_labels:
+            elif lbl in plugin_labels and lbl not in self.auto_always_overwrite:
```
Comment (twaugh) [inline atomic_reactor/plugins/pre_add_labels_in_df.py]: Is there any particular reason you can think of not to just *always* overwrite the labels mentioned in `auto_labels`?

Comment (lcarva) [reply atomic_reactor/plugins/pre_add_labels_in_df.py]: Given the plugin defaults, always overwriting makes a lot of sense.

Comment (rcerven) [reply atomic_reactor/plugins/pre_add_labels_in_df.py]: ok, will always overwrite all auto_labels

**Comment by lcarva** [comment]
@twaugh, waiting to merge this until you approve since you had comments, although, I think they have been addressed.

**Pull request merged**